checkov/bicep/runner.py

```diff
@@ -159,6 +162,9 @@ def add_python_check_results(self, report: Report, runner_filter: RunnerFilter)
                                 record.set_guideline(check.guideline)
                                 report.add_record(record=record)
 
+    def extract_file_path_from_abs_path(self, path: Path) -> str:
+        return f"/{os.path.relpath(str(path), self.root_folder)}"
```

Comment:
```suggestion
        return f"/{os.path.relpath(path, self.root_folder)}"
```
it should work without making it a string 🙂